### ISSUE TITLE
Filtering by dates

### ISSUE THREAD

**Issue by picasocro1** [ppetzold/nestjs-paginate#423]
I would like to use filtering by dates in my pagination.

Any chance to make the below operators
```
FilterOperator.LT,
FilterOperator.LTE,
FilterOperator.GTE,
FilterOperator.GT,
FilterOperator.BTW,
 ```
 working with dates as well?

**Comment by ppetzold** [comment]
Oh, just realized we don't have any coverage for dates. always assumed ISO dates should be working. but if not, happy to accept a PR. otherwise I might have a chance to look into it after the holidays.

**Comment by picasocro1** [comment]
Unfortunately, I don't have anything ready for now. But I can confirm that sadly it doesn't work with dates (I expected ISO dates to be working as well).

Thanks for the reply!

**Comment by picasocro1** [comment]
I've prepared [test cases](https://github.com/ppetzold/nestjs-paginate/pull/426)

**Comment by FrancYescO** [comment]
Maybe it depends on the type of db used, i'm using MariaDB and the filtering with $btw operator is working

**Comment by picasocro1** [comment]
:thinking: I've tested it with MySQL on my project and it wasn't working.

But I've prepared a [pull request with test cases](https://github.com/ppetzold/nestjs-paginate/pull/426) to the repo and these tests show that it's not working correctly though.

**Comment by FrancYescO** [comment]
yep, i've seen the test cases and and using pretty the same syntax in my dev env that's why i've sayd "works for me" :D

**Comment by ppetzold** [comment]
has been fixed with v7.1.2
